indico/modules/events/util.py

```diff
@@ -552,6 +553,39 @@ def serialize_event_for_ical(event, detail_level):
     return data
 
 
+def serialize_event_for_json_ld(event):
+    data = {
+        '@context': 'http://schema.org',
+        '@type': 'Event',
+        'url': event.external_url,
+        'name': event.title,
+        'startDate': event.start_dt_local.isoformat(),
+        'endDate': event.end_dt_local.isoformat(),
+        'location': {
+            '@type': 'Place',
+            'name': event.venue_name or 'No location set',
+            'address': event.address or 'No address set'
+        },
+        'description': strip_tags(event.description),
+    }
+    if event.person_links:
+        data['performer'] = map(serialize_person_for_json_ld, event.person_links)
+    if event.has_logo:
+        data['image'] = event.external_logo_url
+    return data
+
+
+def serialize_person_for_json_ld(person):
+    return {
+        '@type': 'Person',
+        'name': person.display_full_name,
```

Comment:
This should use `full_name` (`display_full_name` takes user preferences into account (whether the first or last name comes first etc) which aren't relevant anyway since search engines won't be logged in)